src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -120,21 +120,20 @@ $range-thumbs-border-active: rem(2px) solid color('indigo');
     border: $range-thumbs-border-error;
   }
 
+  // stylelint-disable selector-max-class
+  // stylelint-disable selector-max-combinators
   // stylelint-disable selector-max-specificity
   &:hover + .Output &,
   &:active + .Output &,
   &:focus + .Output & {
-    transform: translateY(-$range-thumb-size);
-
-    @include page-content-when-not-partially-condensed {
-      transform: translateY(-($range-thumb-size / 2));
-    }
+    transform: translateY(-($range-thumb-size / 2));
```

Comment:
This feels a little far from the button and also it's entirely centered
![screen shot 2019-02-08 at 9 11 28 am](https://user-images.githubusercontent.com/1229901/52483371-bd681000-2b81-11e9-9426-37f363c085a1.png)
:

Reply:
Tooltip is 5px away from its activator, which surprises me because it's not 4-based. Do you know why that distance was decided?

Reply:
Resolved here: https://github.com/Shopify/polaris-react/pull/1019

Reply:
![screen shot 2019-02-11 at 10 09 08 pm](https://user-images.githubusercontent.com/6239480/52615343-c559d580-2e49-11e9-863c-31919eb761ba.jpg)